Src/java/build.gradle

```diff
@@ -129,6 +134,7 @@ configure(subprojects.findAll {it.name in ['model', 'elm', 'quick', 'qdm', 'cql-
         xjc group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
         implementation group: 'org.jvnet.jaxb2_commons', name: 'jaxb2-basics-runtime', version: '1.11.1'
         implementation group: 'jakarta.xml.bind', name: 'jakarta.xml.bind-api', version: '2.3.3'
+        implementation group: 'org.eclipse.persistence', name: 'org.eclipse.persistence.moxy', version: '2.7.7'
```

Comment:
Any chance we can get rid of Moxy here and move to a JAXB module? Android can't import it.

Reply:
Resolved on  https://github.com/cqframework/clinical_quality_language/pull/760/commits/e503ad2096ad72fde5948769106c22e839217a3b